terragrunt/aws/efs/inputs.tf

```
@@ -0,0 +1,9 @@
+variable "aws_security_group_ids" {
+  description = "aws_security_group_ids"
```

nit: could you make these more descriptive.  Maybe something like so?
```suggestion
  description = "AWS security group ID used by the EFS mount target."
```